sdk/python/kfp/v2/compiler/main.py

```diff
@@ -52,7 +55,7 @@ def parse_arguments():
 
 
 def _compile_pipeline_function(pipeline_funcs, function_name, pipeline_root,
-                               output_path, type_check):
+                               pipeline_parameters, output_path, type_check):
```

Comment:
Consider type-hinting this function as well?

Reply:
Done.